<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customersmenu</class>
 <widget class="QDialog" name="customersmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnGetCustomers">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get customers</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableCustomers">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>531</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddNewCustomer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new customer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
